src/java/io/bazel/rulesscala/coverage/instrumenter/JacocoInstrumenter.java

```diff
@@ -69,6 +70,11 @@ private void processArg(Instrumenter jacoco, String arg) throws Exception {
                     throw new RuntimeException(e);
                 }
             });
+
+            Files.write(
+                outFS.getPath("-paths-for-coverage.txt"),
```

Comment:
maybe add a link to the bazel doc or jacoco doc (or both) that explain about this feature?

Reply:
Ha! Unfortunately, this is fairly undocumented in bazel / bazel's implementation of jacoco. I updated the comments to reflect this as best I could. Let me know if I should add some more color.

Reply:
This is awesome, thanks!
I think one more thing we should do (I can do it if you prefer) is to open a bazel issue for documentation m. The main reason I want this is to have them acknowledge this is a public API and not a hack they will break.

Reply:
Filed in: https://github.com/bazelbuild/bazel/issues/10857